utility/test/gov_test.go

```diff
@@ -87,7 +87,7 @@ func TestUtilityContext_GetBlocksPerSession(t *testing.T) {
 	ctx := NewTestingUtilityContext(t, 0)
 	defaultParams := DefaultTestingParams(t)
 	defaultParam := int(defaultParams.GetBlocksPerSession())
-	gotParam, err := ctx.GetParameter(typesUtil.BlocksPerSessionParamName, 0)
+	gotParam, err := ctx.GetBlocksPerSession()
```

Comment:
I thought it was weird that all the other parameters had their own wrapper around a `GetParameter()` call and this was the only one without it - I will change it back but I thought it would be better to fit the pattern and just make the function a wrapper like the other parameters

Reply:
Sgtm. Let's keep it